core/cli/src/tasks.ts

```diff
@@ -97,7 +90,7 @@ export async function runTasksFromConfig(
     for (const task of tasks) {
       try {
         logger.info(styles.taskHeader(`running ${styles.task(task.id)} task`))
-        await task.run({ files, command, cwd: config.root })
+        await task.run({ files, command, cwd: config.root, config })
```

Comment:
**thought**: this is going to lead to future crimes i bet

Reply:
yeah that was my worry too. if you can think of another way of achieving this i'm all ears!

Reply:
do you think we should `Object.freeze` the object before we pass it here to minimise the crime potential?

Reply:
and make it `readonly` in the types